backend/services/AccountWebhookService.ts

```diff
@@ -0,0 +1,74 @@
+import { Client as AccountServiceWebhookClient } from '../types/openapi-account-service';
+import { OpenAPIClientAxios, type Document } from 'openapi-client-axios';
+import openapiJson from '../openapi-specs/account-service.json' assert { type: 'json' };
+import * as Config from '../config/config';
+import { HttpStatusCode } from 'axios';
+import { Response } from 'express';
+import logger from '../logger';
+import { WebhookController } from '../controllers/WebhookController';
+import { HttpError } from '../types/HttpError';
+
+type AccountServiceWebhookResponse = {
+  referenceId: string;
+  handle: string;
+  msaId: string;
+  accountId: string;
+};
```

Comment:
Let's create an issue to put the webhook request & response payload types into an OpenAPI spec document in the `account-service` repo, copy to this repo & generate types from it. (I can show you what I did in `content-watcher` for the same)

Reply:
Cool, there's already an openApi spec in `account-service/docs/webhook.endpoints.json` 
#44